code/modules/cargo/supplypacks/contraband.dm

```diff
@@ -195,3 +195,30 @@
 	container_type = /obj/structure/closet/crate
 	container_name = "teutonic armor crate"
 	contraband = 1
+
+/datum/supply_pack/robotics/salvage
+	name = "Redirected Mech Salvage"
+	contains = list(/obj/item/mechasalvage = 6)
+	cost = 400 //High End Mech Parts, Direct To You, at an extreme markup!
+	container_type = /obj/structure/closet/crate
+	container_name = "oil-stained crate"
+	contraband = 1
+
+/datum/supply_pack/randomised/material/rareores
+	name = "Smuggled Ores and Materials"
+	num_contained = 20
+	contains = list(
+		/obj/item/ore/vaudium,
```

Comment:
Honestly, none of these ore are particularly valuable enough that you need to mix both raw ore and processed sheets, and can really just make these ore into their processed sheet forms (even if that effectively doubles the current spawn rate for these materials)